bindings/php/opendal-php.stubs.php

```diff
@@ -20,6 +20,111 @@
 
 // Stubs for opendal-php
 
-namespace {
-    function debug(): string {}
+namespace OpenDAL {
+    class Operator {
+        public function __construct(string $scheme_str, array $config) {}
+
+        /**
+         * Write bytes into given path.
+         */
+        public function write(string $path, string $content): mixed {}
+
+        /**
+         * Read the whole path into bytes.
+         */
+        public function read(string $path): string {}
+
+        /**
+         * Check if this path exists or not.
+         */
+        public function is_exist(string $path): int {}
+
+        /**
+         * Get current path's metadata **without cache** directly.
+         *
+         * # Notes
+         *
+         * Use `stat` if you:
+         *
+         * - Want detect the outside changes of path.
+         * - Don't want to read from cached metadata.
+         */
+        public function stat(string $path): \OpenDAL\Metadata {}
+
+        /**
+         * Delete given path.
+         *
+         * # Notes
+         *
+         * - Delete not existing error won't return errors.
+         */
+        public function delete(string $path): mixed {}
```

Comment:
What's this `mixed` for?

Reply:
The return value here is actually void, which means there is no return. This file was generated by `cargo php stubs`. In PHP, mixed refers to any type and I don't know why cargo-php generates mixed here.

Reply:
Ok, thanks!